### PR TITLE
Fix currency formatting in the in game console

### LINKED ISSUE #23286: Console reports wrong money when set to very negative number
### Operating System

Windows 11

### OpenRCT2 build

v 0.4.16

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This is a development issue

### Describe the issue

Use the console command
`set money -92999990866648780.8`

However, the console reports
`money 1145528628.-80`

The actual money is not the value reported. This is probably a 64 bit vs 32 bit conversion issue.

### Steps to reproduce

See description

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by mrmbernardi**
Fixes #23286

### CODE REVIEW

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 1]
```diff
@@ -559,32 +559,35 @@ static int32_t ConsoleCommandGet(InteractiveConsole& console, const arguments_t&
...
-            console.WriteFormatLine("max_loan %d", gameState.MaxBankLoan / 10);
+            console.WriteLine(FormatString("max_loan {CURRENCY2DP}", gameState.MaxBankLoan));
         }
         else if (argv[0] == "guest_initial_cash")
         {
-            console.WriteFormatLine(
-                "guest_initial_cash %d.%d0", gameState.GuestInitialCash / 10, gameState.GuestInitialCash % 10);
+            console.WriteLine(FormatString("guest_initial_cash {CURRENCY2DP}", gameState.GuestInitialCash));
+        }
+        else if (argv[0] == "construction_rights_cost")
```
Comment (Gymnasiast) [inline src/openrct2/interface/InteractiveConsole.cpp:588]: Why did you move this block up? It seems to me that keeping it together with "land price" makes more sense.

Comment (mrmbernardi) [reply src/openrct2/interface/InteractiveConsole.cpp:588]: Just to keep the currencies together which helped when comparing diffs. I can put it back if you want.

Comment (mrmbernardi) [reply src/openrct2/interface/InteractiveConsole.cpp:588]: Oh, just noticed I missed one anyway

Comment (mrmbernardi) [reply src/openrct2/interface/InteractiveConsole.cpp:588]: I moved both of them up.

Comment (Gymnasiast) [reply src/openrct2/interface/InteractiveConsole.cpp:588]: Could you put "land_rights_cost" above "construction_rights_cost"?  This is how they’re usually ordered. (Forgive me my high-octane autism here.)

Comment (mrmbernardi) [reply src/openrct2/interface/InteractiveConsole.cpp]: Done :)

**Review by Gymnasiast — APPROVED** [review]

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
Needs changelog, LGTM otherwise.

**Comment by mrmbernardi** [comment]
Added the changelog entry

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**